docs/guidance/map/connecting_publications.md

```diff
@@ -0,0 +1,429 @@
+
+Unless publications have disjoint coverage of OCDS fields or implement consistent mappings, reusing the ocids across publications can result in unpredictable and nonsensical results when releases are merged to create a compiled release. For example, unless publications use an identical approach to assigning [local identifiers](../../schema/identifiers.md#local-identifiers) to objects in arrays, then awards, contracts and other objects in arrays can overwrite and/or duplicate each other in nonsensical ways.
+
+### Example: Reusing ocids across Public Contracts Scotland and Find a Tender Service
+
+The PCS and FTS publications have many overlapping fields, but implement slightly different mappings. Therefore, if FTS were to reuse the ocids minted by PCS, merging releases would result in nonsensical data
+
+For example, the publications implement inconsistent mappings for the `.id` of the buyer in the `.parties` array so merging releases would result in duplicate buyers:
+
+`````{tab-set}
+````{tab-item}  PCS release
+```json
+{
+  "id": "rls-1-JAN468109",
+  "ocid": "ocds-r6ebe6-0000710562",
+  "date": "2023-01-09T00:00:00Z",
+  "tag": [
+    "planning"
+  ],
+  "parties": [
+    {
+      "id": "org-1",
+      "identifier": {
+        "legalName": "Fife Council"
+      },
+      "name": "Fife Council",
+      "roles": [
+        "buyer"
+      ]
+    }
+  ]
+}
+```
+````
+````{tab-item}  FTS release
+```json
+{
+  "id": "000521-2023",
+  "ocid": "ocds-r6ebe6-0000710562",
+  "date": "2023-01-09T12:26:29Z",
+  "tag": [
+    "planning"
+  ],  
+  "parties": [
+    {
+      "id": "GB-FTS-2273",
+      "identifier": {
+        "legalName": "Fife Council"
+      },
+      "name": "Fife Council",
+      "roles": [
+        "buyer"
+      ]
+    }
+  ]
+}
+```
+````
+````{tab-item}  Compiled release
+```json
+{
+  "ocid": "ocds-r6ebe6-0000710562",
+  "parties": [
+    {
+      "id": "org-1",
+      "identifier": {
+        "legalName": "Fife Council"
+      },
+      "name": "Fife Council",
+      "roles": [
+        "buyer"
+      ]
+    },
+    {
+      "id": "GB-FTS-2273",
+      "identifier": {
+        "legalName": "Fife Council"
+      },
+      "name": "Fife Council",
+      "roles": [
+        "buyer"
+      ]
+    }
+  ]
+}
+```
+````
+`````
+
+## How to connect releases from different publications without reusing ocids
+
+If publications have overlapping fields and inconsistent mappings, then you ought to use [links](../../schema/reference.md#link) to connect releases across the publications. If, like FTS, a publication's releases are derived from another publication's data, the derivative publication ought to use the 'canonical' [link relation type](../../schema/codelists.md#link-relation-type) to refer to the releases in the authoritative publication. If a publication's releases follow another publication's releases, the subsequent publication ought to use the 'prev' link relation type to refer to the earlier releases.
```

Comment:
Just one thought here. In the case of 'canonical' releases it's clear which publisher should include the `links` object. However in the case of 'prev' releases it's less clear. I can imagine the following sort of case (in chronological order):

1. publisher A publishes a tender release
2. publisher B then publishes a tender release for the same process and `links` to publisher A's release as 'prev'
3. publisher A then publishes an award release for the same process

Does publisher A now need to reference publisher B's release as it was published before publisher A's award release? Maybe I'm over thinking this but from this advice I think if I was a publisher this is something I might not be clear on.

Reply:
In that scenario, I would expect publisher B's tender release to refer to publisher A's original tender release using the 'canonical' link relation type. I don't think publisher's A's award release would need to link to publisher B's tender release as that would require publisher A's system to be aware of publisher B's system.

I'll leave this comment thread open in case @jpmckinney thinks differently.

Reply:
I haven't read the PR yet, but the 'prev' code was described in #928 as:

> The linked resource is a previous publication about this contracting process, that is not in OCDS format or is using a different OCID. This code is intended for use when transitioning from an old system without converting its data, which might occur in cases of jurisdictional changes like municipal mergers.

#1326 landed on the following, based on https://www.iana.org/assignments/link-relations/link-relations.xhtml:

> Indicates that a release or compiled release is a part of a series, and that the previous in the series is the link target.

That description does allow for extremely complicated implementations – but, really, it was designed for only the simple case where a jurisdiction has transitioned from an old system to a new system, and it needs to link to the previous "release" of information about the contracting process.

As I said, I haven't read the PR. If the PR doesn't make this specific use case clear (or describes broader uses of 'prev'), then it should be simplified to only describe this use case for which we have evidence.

Reply:
@jpmckinney apologies, I had somehow missed replying to this comment. 

The PR doesn't describe the specific use case of referring back to a non-OCDS publication because the scope of #1430 is about linking releases from different OCDS publications.

It has the following to say about 'prev':

> If a publication's releases follow another publication's releases, the subsequent publication ought to use the 'prev' link relation type to refer to the earlier releases.

Given the above and having reviewed the PR, does your comment still stand?

If so, I guess we could broaden the scope of the page and a add section along the lines of "How to refer to a non-OCDS publication".

Reply:
Okay, I'll review in full.

Reply:
I've reviewed, and I just added a note admonition about the use of 'prev' in the context of non-OCDS admonitions.